src/Centreon/Infrastructure/HostConfiguration/Repository/HostConfigurationRepositoryRDB.php

```diff
@@ -866,4 +868,30 @@ private function updateMonitoringServerRelation(int $hostId, int $monitoringServ
         $statement->bindValue(':host_id', $hostId, \PDO::PARAM_INT);
         $statement->execute();
     }
+
+    /**
+     * Add or update the link between a host and a host severity
+     *
+     * @param Host $host
+     */
+    private function updateHostSeverity(Host $host): void
+    {
+        if (empty($host->getSeverities())) {
+            return;
+        }
+
+        $request = $this->translateDbName(
+            "DELETE `:db`.hostcategories_relation
+            FROM `:db`.hostcategories_relation
+            JOIN `:db`.hostcategories
+            ON hostcategories.hc_id = hostcategories_relation.hostcategories_hc_id
+            WHERE hostcategories.level IS NOT NULL
+            AND hostcategories_relation.host_host_id = :host_id"
+        );
+        $statement = $this->db->prepare($request);
+        $statement->bindValue(':host_id', $host->getId(), \PDO::PARAM_INT);
+        $statement->execute();
+
+        $this->linkSeveritiesToHost($host);
```

Comment:
why method is named with plural
host can have only 1 severity, isn't it ?

Reply:
As of today, the method add as many severity to the host as returned by $host->getSeverities() witch returns an array.

But you're right, I signaled this inconsistency to omercier as I found two different ways to link multiple severities to the same host, it will be fixed in another ticket

Reply:
maybe a misunderstanding between categories and severities

Reply:
Modifications has been made to handle a unique severity